### ISSUE TITLE
Lỗi mất CTC Không tìm thấy đường dẫn hợp lệ

### ISSUE THREAD

**Issue by conqueror7** [pbcquoc/vietnamese_ocr#1]
Chào,
Tôi đang nhận được lỗi dưới đây. Xin hãy giúp tôi giải quyết nó.
Epoch 1/100
2019-06-19 11:56:44.007582: I tensorflow/stream_executor/dso_loader.cc:153] successfully opened CUDA library libcublas.so.10.0 locally
667/667 [==============================] - 1082s 2s/step - loss: inf - val_loss: 134.201341831: W tensorflow/core/util/ctc/ctc_loss_calculator.cc:144] No valid path found.
